lang/de_DE.po

```diff
@@ -2465,10 +2467,10 @@ msgid "States"
 msgstr "Status"
 
 msgid "Reset states"
-msgstr ""
+msgstr "Status zurücksetzen"
 
 msgid "Current total state count"
-msgstr ""
+msgstr "Gesamtanzahl an Status"
```

Comment:
Das weiß ich, ist aber so korrekt.
Alternativ muss der ganze String geändert werden.
Wir könnten es in ein Wort packen: Gesamtstatusanzahl